front/app/components/HomeProjectList.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import ProjectDetail from './ProjectDetail'
+
+class HomeProjectList extends React.Component {
+
+    constructor() {
```

Comment:
Je n'ai encore jamais fait de composants en mode class du coup je vais peut être te dire de la merde (et si c'est le cas je veux bien que tu m'explique un peu :) )

J'ai l'impression que ce constructor est inutile. 
le this.state devrait être dans une méthode du cycle de vie des composants React qui est "getInitialState" et qui renverait { projects : [] }.
En règle général on ne modifie jamais directement le state, soit on passe par getInitialState pour l'état initial soit par la méthode this.setState(stateObject) qui va entre autre relancer un render (si l'état change le composant se réaffiche pour se mettre à jour)

Je ne comprend pas l'utilité du bien sur this.render